src/deps/miniasync/utils/docker/build.sh

```diff
@@ -24,7 +24,7 @@ source $(dirname $0)/valid-branches.sh
 
 if [[ -z "$OS" || -z "$OS_VER" ]]; then
 	echo "ERROR: The variables OS and OS_VER have to be set " \
-		"(eg. OS=fedora, OS_VER=30)."
+		"(eg. OS=fedora, OS_VER=37)."
```

Comment:
Suggests not changing /deps via PR in pmdk.
First of all, the change should be in the miniasync repo and then the entire deps folder updated.
If I am not mistaken in this case this change does nothing anyway.